src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,12 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.8.0
+TBD
+
+### Breaking changes
+- Default value for `.Values.buildkit.service.sessionAffinity` is now empty. For cluster updates, please refer to the latest self-hosted guide to set this value for your own installation.
```

Comment:
Could you document `buildkit.service.sessionAffinity` and link the docs from here? I wouldn't refer to the cloud provider guides. I would say that if you want to keep using `ClientIP` session affinity, you need to do:
```
buildkit:
  service:
     sessionAffinity: ClientIP
```